proposals/0337-support-incremental-migration-to-concurrency-checking.md

```diff
@@ -268,7 +268,7 @@ This proposal should not otherwise affect ABI.
 
 ### A "concurrency epoch"
 
-If the evolution of a given module is tied to a version that can be expressed in `@available`, it is likely that there will be some specific version where it retroactively adds concurrency annotations to its public APIs, and that thereafter any new APIs will be "born" with correct concurrency annotations. We could take advantage of this by allowing the module to specify a "concurrency a particular version when it started ensuring that new APIs were annotated and automatically applying `@preconcurrency` to APIs available before this cutoff.
+If the evolution of a given module is tied to a version that can be expressed in `@available`, it is likely that there will be some specific version where it retroactively adds concurrency annotations to its public APIs, and that thereafter any new APIs will be "born" with correct concurrency annotations. We could take advantage of this by allowing the module to specify a particular version when it started ensuring that new APIs were annotated and automatically applying `@preconcurrency` to APIs available before this cutoff.
```

Comment:
@DougGregor @beccadax I wasn't quite sure what this was supposed to say. Maybe you mean to say something like this instead?

> … to specify a "concurrency version", i.e. a particular version when …

Reply:
Yeah, this is meant to say that we could say "module X adopted concurrency in version Y", and use that as a blanket statement to put `@preconcurrency` on any declaration with availability less than version Y.